include/sisl/grpc/rpc_server.hpp

```diff
@@ -41,15 +41,17 @@ class GrpcServer : private boost::noncopyable {
                const std::string& ssl_cert);
     GrpcServer(const std::string& listen_addr, uint32_t threads, const std::string& ssl_key,
                const std::string& ssl_cert, const std::shared_ptr< sisl::GrpcTokenVerifier >& auth_mgr);
+    GrpcServer(const std::string& listen_addr, uint32_t threads, int max_receive_msg_size, const std::string& ssl_key,
+               const std::string& ssl_cert, const std::shared_ptr< sisl::GrpcTokenVerifier >& auth_mgr);
     virtual ~GrpcServer();
 
     /**
      * Create a new GrpcServer instance and initialize it.
      */
     static GrpcServer* make(const std::string& listen_addr, uint32_t threads = 1, const std::string& ssl_key = "",
-                            const std::string& ssl_cert = "");
+                            const std::string& ssl_cert = "", int max_receive_msg_size = 0);
```

Comment:
i suggest to set the default value of max_receive_msg_size to 64M, 0 is easy to be misused

Reply:
0 means default of grpc(will not set the value).  Which is 4M.

The SISL is not exclusive for nuobject 2.0 so setting default to 64 is not ideal